src/Evolution/Systems/ScalarTensor/Sources/ScalarSource.cpp

```diff
@@ -0,0 +1,25 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/ScalarTensor/Sources/ScalarSource.hpp"
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace ScalarTensor {
+
+void add_scalar_source_to_dt_pi_scalar(
```

Comment:
are you sure you need this function?

Reply:
Wrote it to have parallel structure with the stress tensor part